app/controllers/messages_controller.rb

```diff
@@ -19,6 +19,10 @@ def new
   def create
     @message = Message.new(message_params)
 
+    if !verify_recaptcha(model: @user)
+      (redirect_to "/users/#{current_user.id}#messages") && return
```

Comment:
I'd put the `return` on its own line. Otherwise, it's dependent on what the `redirect_to` returns.

Reply:
Okay, gave that a go!